content/en/real_user_monitoring/mobile_and_tv_monitoring/android/setup.md

```diff
@@ -417,8 +417,31 @@ OkHttpClient okHttpClient = new OkHttpClient.Builder()
 {{% /tab %}}
 {{< /tabs >}}
 
+If you want RUM resources and Spans to be automatically created for your OkHttp requests, you can use the `DatadogInterceptor` as a network interceptor.
```

Comment:
There is a bit of confusion here by using `network interceptor`, because of the sentence `In case you want to monitor the network redirects or retries, you can use the DatadogInterceptor as a network interceptor.` below.

Is it meant for the `networkInterceptor` usage here or `interceptor`?

Reply:
ahh mistake, it should be `interceptor` there